webapp/WEB-INF/templates/admin/system/manage_files_system.html

```diff
@@ -14,7 +14,7 @@
 						<td>${file.name}</td>
 						<td>${file.description}</td>
 						<td>
-							<@aButton href='jsp/admin/system/ManageFilesSytemDir.jsp?dir=${file.directory}' buttonIcon='eye' title='#i18n{portal.system.files_system.view} ${file.name}' showTitle=false />
+							<@aButton href='jsp/admin/system/ManageFilesSytemDir.jsp?dir=${file.directory}' buttonIcon='eye' title='#i18n{portal.system.manage_files_system.vView} ${file.name}' showTitle=false />

```

Comment:
Seems like this new key has a typo: 
src/java/fr/paris/lutece/portal/resources/system_messages.properties:manage_files_system.view=View
src/java/fr/paris/lutece/portal/resources/system_messages_fr.properties:manage_files_system.vView=Voir

but it looks like it's the correct one to use